### PR TITLE
Code for populating table added

### PR DESCRIPTION
**Pull request by Sakshisak**
Code for Populating users table

Updating gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 __pycache__/
 /lifeiitkbackend/__pycache__
+/secrets.yml
```
Comment (ditttu) [inline .gitignore]: Why is this here?

**Review by ditttu — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (Sakshisak) [reply .gitignore]: We don't want others to see this right?
